CesiumGltf/include/CesiumGltf/AccessorUtility.h

```diff
@@ -229,6 +229,28 @@ struct IndicesForFaceFromAccessor {
   int32_t primitiveMode;
 }; // namespace CesiumGltf
 
+/**
+ * Visitor that retrieves the vertex index from the given accessor type as an
+ * int64_t. This should be initialized with the index of the vertex index.
```

Comment:
Hm is there a clearer way to word "index of the vertex index"? 
```suggestion
 * int64_t. This should be initialized with the index (within the 
 * accessor itself) of the vertex index.
```